src/main/webapp/index.html

```diff
@@ -18,8 +18,7 @@
 			<div id="site-title">Givr</div>
 			<div id="nav-links">
 				<a href="index.html">Get Help</a>
-				<a href="registration.html">Register Organization</a>
```

Comment:
Why is the registration being removed?

Reply:
It should have been removed when we implemented the rebuild nav bar method. Registration option is only shown after the user has logged in